skimage/morphology/isotropic.py

```diff
@@ -9,7 +9,7 @@
 def isotropic_erosion(image, radius, out=None, spacing=None):
     """Return binary morphological erosion of an image.
 
-    This function returns the same result as :func:`skimage.morphology.binary_erosion`
+    This function returns the same result as :func:`skimage.morphology.erosion`
```

Comment:
If it actually returns the same result as `binary_erosion`, when changing to `erosion`, I guess we need to include the same warning that differentiates `erosion` from `binary_erosion`, i.e., "Note the pixel shift by 1 for even-sized footprints (see docstring notes)."

Reply:
Maybe, after the 'Returns' section:
```py
    Notes                                                                      
    -----                                                                     
    For even-sized footprints, :func:`skimage.morphology.erosion` and
    this function produce an output that differs: one is shifted by one pixel
    compared to the other. :func:`skimage.morphology.pad_footprint´ is available
    to account for this.
```
?

Reply:
Hmm, I'm not sure that the docstrings claim "the same result" makes a lot of sense in the first place. This function takes a `radius` to determine the neighborhood while both other `erosion` functions take in a `footprint`. So `isotropic_erosion` seems to be limited to a circular footprint of uneven size!

So we should probably reword this. What do you think?

Reply:
See https://github.com/scikit-image/scikit-image/pull/7665/commits/549dc4732aee767b83f02ab49deb1c4595574c13 as a sugggestion.

Reply:
You're right; I skimmed instead of glancing over this... I very much like 549dc4732aee767b83f02ab49deb1c4595574c13; thanks for this. I just made a few suggestions (latest review). Also, when we merge, I'd like to remember to change "uneven size" into "odd size" in the PR description.